packages/fxa-settings/package.json

```diff
@@ -69,6 +70,7 @@
     "eslint-plugin-react": "^7.19.0",
     "fxa-shared": "workspace:*",
     "node-sass": "^4.14.0",
+    "nodemon": "^2.0.4",
```

Comment:
where are we using this?

Reply:
whoops, this was from an experiment

Reply:
Was this to run `postcss` when the tailwind config or tailwind.css changes? I feel like you could sneak that in here too, or it could be done in a follow up (it's been pretty annoying for me to manually run, didn't set that up in my branch in case you were doing that her)

Reply:
Actually, that task would need to watch and recompile at any scss changes as well to a accommodate what I've got in my open PR with those custom files/classes. We can make a follow up issue at an est of a 1 or 2 if that can't make it in here for the sprint.

Reply:
Yeah, so I've got these set up in the package.json:

```
    "poststart": "watch-postcss",
    "postrestart": "watch-postcss",
    "watch-postcss": "nodemon --watch ./src/tailwind.css --exec npm run build-postcss"
```

Though it's not currently working. I also suspect this would have issues as it doesn't know when the PM2 instance gets killed. And as you mentioned, it doesn't work with _any_ SCSS files, only the main Tailwind file. How this would work with SCSS files (especially external component SCSS files), without also conflicting with Hot Module Reloading that watches these files already, I'm not sure,

Reply:
I'll make a followup issue for this.